phpBB/styles/prosilver/theme/content.css

```diff
@@ -361,6 +361,15 @@ dd.option {
 	line-height: 1.4em;
 }
 
+.agreement {
+	font-size: 10px;
+}
+
+.agreement .agreement-text {
```

Comment:
this should only need to be .agreement-text no need to over scope here? or is prosilver preventing this?

Reply:
Why not just: `.agreement p { ... }` ?

Reply:
because you are over qualifying the selector. We do not allow directly targeting html elements as it leads to bad inheritance. Always use classes

Reply:
But it won't have effect in this case, because `.content p` and `.panel p` overwrite it.